config/settings/development.yml

```
@@ -19,6 +19,7 @@ nhs_api:
 pds:
   enqueue_bulk_updates: false
   rate_limit_per_second: 5
+  enabled: true
```

```suggestion
  enabled: false
```

This should be `false` for development, and then the default value in `settings.yml` should have it set to `true`.